### ISSUE TITLE
Docker connection fails with ReplyError: ERR unknown command `graph.query`

### ISSUE THREAD

**Issue by machinaeXphilip** [RedisGraph/redisgraph.js#79]
I am trying to get redisgraph.js to work with the redisgraph docker container on Mac OS 11.4 .

I started the container with ` docker run -p 6379:6379 -it --rm redislabs/redisgraph`

It runs as expected 
```bash
1:M 05 Aug 2021 20:48:43.818 # Server initialized
1:M 05 Aug 2021 20:48:43.826 * <graph> Starting up RedisGraph version 2.4.7.
1:M 05 Aug 2021 20:48:43.842 * <graph> Thread pool created, using 2 threads.
1:M 05 Aug 2021 20:48:43.842 * <graph> Maximum number of OpenMP threads set to 2
1:M 05 Aug 2021 20:48:43.842 * Module 'graph' loaded from /usr/lib/redis/modules/redisgraph.so
1:M 05 Aug 2021 20:48:43.843 * Ready to accept connections
```

I can log into the docker container and use it's redis-cli:

```bash
127.0.0.1:6379> GRAPH.QUERY MotoGP "CREATE (:Rider {name:'Valentino Rossi'})-[:rides]->(:Team {name:'Yamaha'}), (:Rider {name:'Dani Pedrosa'})-[:rides]->(:Team {name:'Honda'}), (:Rider {name:'Andrea Dovizioso'})-[:rides]->(:Team {name:'Ducati'})"
1) 1) "Labels added: 2"
   2) "Nodes created: 6"
   3) "Properties set: 6"
   4) "Relationships created: 3"
   5) "Cached execution: 0"
   6) "Query internal execution time: 7.312416 milliseconds"
127.0.0.1:6379> 
```

but using redisgraph.js on the host machine fails:

```js
const RedisGraph = require("redisgraph.js").Graph;

let graph = new RedisGraph("social");

(async () =>{
    await graph.query("CREATE (:person{name:'roi',age:32})");   
})();

```
throws the error:

```bash
node:3433) UnhandledPromiseRejectionWarning: ReplyError: ERR unknown command `graph.query`, with args beginning with: `social`, `CREATE (:person{name:'roi',age:32})`, `--compact`, 
```

I must be misunderstanding how the exposing of the ports work (-p 6379:6379 I though should expose the docker containers internal 6379 and map it to the hosts machines localhost:6379) or am I missing sth else?

The error itself makes me get the impression I am connecting successfully to a redis-server that has the redisgraph module not included. But as far as I can tell, I should only have the redis-server inside of the docker container running. When stopping the docker container, the error message does not change, which supports this theory, but there is no other redis-server on this machine (uninstalled all some time ago)

**Comment by leibale** [comment]
To prove your theory just try to use a different port:
```bash
docker run -p 6379:1234 -it --rm redislabs/redisgraph
```
```javascript
const graph = new RedisGraph('social', 'localhost', 1234);
```

If you want to find the "unknown" redis-server you can use:
```bash
ps -ef | grep redis-server
```

If you want to shut it down you can use the `SHUTDOWN` command:
```bash
redis-cli
> SHUTDOWN
```

**Comment by machinaeXphilip** [comment]
thank you! that helped.

i was wrong. there was still a redis-server process running although I had removed redis-server from my machine. Must have removed it while running without shutting it down the server before. So the server process was still running on my machine and blocking the port without redis-server being installed. 
For I had also deinstalled redis-cli, I could not shutdown via it. 

But simply restarted my machine. 🤦

Awkward, but it was a simple "have to tried to restart your computer" scenario.
